docs/api/browser-window.md

```diff
@@ -467,23 +467,23 @@ The `BrowserWindow` class has the following static methods:
 
 #### `BrowserWindow.getAllWindows()`
 
-Returns an array of all opened browser windows.
+Returns `BrowserWindow[]` - An array of all opened browser windows.
```

Comment:
Returns an array of BrowserWindow instances that correspond to all the existing browser windows.